src/main/resources/assets/milky/recipes/bucket_chocolate_milk.json

```diff
@@ -0,0 +1,16 @@
+{
+    "type": "milky:milk_based",
+    "ingredients": [
+        {
+			"item": "minecraft:milk_bucket"
```

Comment:
This should be the Forge Universal Bucket containing the mod's Milk fluid, not the vanilla Milk Bucket item.

Reply:
done